File: WargamePro/Order.kt
package com.example.wargamesimulatorpro

import android.os.*
import kotlinx.parcelize.Parcelize

@Parcelize
data class Order(
    var orderCost: Int,
    val orderName: String,
    val availablePhase: String,
    val targetsEnemy: Boolean,
    val description: String
) : Parcelable

File: C#Scripts/Data/Order.cs
using System;

public class Order
{
    public int OrderCost;
    public string OrderName;
    public string AvailablePhase;
    public bool TargetsEnemy;
    public string Description;

    public Order() { }

    public Order(int orderCost, string orderName, string availablePhase, bool targetsEnemy, string description)
    {
        OrderCost = orderCost;
        OrderName = orderName;
        AvailablePhase = availablePhase;
        TargetsEnemy = targetsEnemy;
        Description = description;
    }
}
